doc/API.md

```diff
@@ -960,6 +960,192 @@ sql.getBounds('select * from table').done(function(bounds) {
+
+cartodb.Image(vizjson_url, {override_bbox: true})
+  .size(600, 400)
+  .center([-3.4, 44.2])
+  .zoom(4)
+  .write({ class: "thumb", id: "AwesomeMap" });
+</script>
+```
+
+#### Result
+```html
+<img id="AwesomeMap" src="https://cartocdn-ashbu.global.ssl.fastly.net/documentation/api/v1/map/static/center/04430594691ff84a3fdac56259e5180b:1419270587670/4/-3.4/44.2/600/400.png" class="thumb">
+```
+
+#### cartodb.Image(_layerSource_, [options])
+
+##### Arguments
+
+- **layerSource**: can be either a viz.json object or a [layer source object](http://docs.cartodb.com/cartodb-platform/cartodb-js.html#standard-layer-source-object-type-cartodb)
+
+##### Options
+
+Options take the form of a JavaScript object.
+
+- **options**:
+    - **basemap**: change the basemap specified in the layer definition. Type: Object defining base map properties (see example below).
+    - **no_cdn**: Disable CDN usage. Type: Boolean. Default: `false` (use CDN)
+    - **override_bbox**: Override default of using the bounding box of the visualization. This is needed to use `Image.center` and `Image.zoom`. Type: Boolean. Default: `false` (use bounding box)
+
+```javascript
+<script>
+var vizjson_url = 'https://documentation.cartodb.com/api/v2/viz/008b3ec6-02c3-11e4-b687-0edbca4b5057/viz.json';
+var basemap = {
+        type: "http",
+        options: {
+          urlTemplate: "http://{s}.basemaps.cartocdn.com/dark_all/{z}/{x}/{y}.png",
+          subdomains: ["a", "b", "c"]
+        }
+      };
+
+cartodb.Image(vizjson_url, {override_bbox: true, basemap: basemap})
+  .size(600, 400)
+  .center([0,0])
+  .write({ class: "thumb", id: "AwesomeMap" });
+</script>
+```
+
+##### Returns
+An _Image_ object
+
+### cartodb.Image
+
+#### Image.size(_width_,_height_)
+
+Sets the size of the image.
+
+##### Arguments
+
+- **width**: the width of the resulting image in pixels
+- **height**: the height of the resulting image in pixels
+
+##### Returns
+An _Image_ object
+
+#### Image.center(_latLng_)
+
+Sets the center of the map. Must be used with the option `override_bbox: true`.
+
+##### Arguments
+
+- **latLng**: an array of the latitude and longitude of the center of the map. Example: `[40.4378271,-3.6795367]`
+
+##### Returns
+
+An _Image_ object
+
+#### Image.zoom(zoomLevel)
+
+Sets the zoom level of the static map. Must be used with the option `override_bbox: true`.
+
+##### Arguments
+
+- **zoomLevel**: the zoom of the resulting static map. `zoomLevel` must be an integer in the range [0,24].
+
+##### Returns
+
+An _Image_ object
+
+#### Image.bbox(_boundingBox_)
+
+If you set `bbox`, `center` and `zoom` will be overridden. Must be used with the option `override_bbox: true`
+
+##### Arguments
+
+- **boundingBox**: an array of coordinates making up the bounding box for your map. `boundingBox` takes the form: `[sw_lat, sw_lon, ne_lat, ne_lon]`.
+
+##### Returns
+
+An _Image_ object
+
+#### Image.into(HTMLImageElement)
+
+Inserts the image into the HTML DOM element specified.
```

Comment:
I think we could add a small example here.

For example something like this:
https://github.com/CartoDB/cartodb.js/wiki/CartoDB-Map-API#imageintohtmlimageelement